pegasusio/zarr_utils.py

```diff
@@ -244,6 +255,7 @@ def read_unimodal_data(self, group: zarr.Group) -> UnimodalData:
                             feature_multiarrays = self.read_mapping(group['feature_multiarrays']),
                             barcode_multigraphs = self.read_mapping(group['barcode_multigraphs']) if 'barcode_multigraphs' in group else dict(), # for backward-compatibility
                             feature_multigraphs = self.read_mapping(group['feature_multigraphs']) if 'feature_multigraphs' in group else dict(), # for backward-compatibility
+                            img = self.read_dataframe(group['img']) if 'img' in group else dict(), # for backward-compatibility
```

Comment:
@rocherroche This only works for the case of `SpatialData`. For all other DataClass, it will break because they don't have `img` field. This is why your CI test failed.